### DISCUSSION TITLE
Alfen Wallbox firmware version

### DISCUSSION THREAD

**Discussion by Pd9040** [leeyuentuen/alfen_wallbox#97] [Q&A] [answered]
For which firmware version of the Alfen Wallbox is the HA integration made? Will the integration work with the latest firmware? NG9xx 6.5.0-4217-BL
12-10-2023

**Comment by leeyuentuen** [comment, marked as answer]
yes it should work on that firmware, everything above 6 should work.

**Reply by Pd9040** [reply]
Thank you, will test :-)
